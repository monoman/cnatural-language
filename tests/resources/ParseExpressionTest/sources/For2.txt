() => {
	for (var v = 0;;)
		v++;
}