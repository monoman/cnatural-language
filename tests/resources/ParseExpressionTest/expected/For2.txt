Lambda( => {
for (var v = Literal(DecimalInteger, [0]);
Condition();
)
(PostIncrement Name(v));
}
)